File: 프로그래머스/0/181885. 할 일 목록/할 일 목록.js
function solution(todo_list, finished) {
       var count=0;
        for(var i=0; i<finished.length ; i++){
            if(!finished[i]) {
                count++;
            }
        }
        var answer = [count];
        var n=0;
        for(var i=0; i<todo_list.length;i++){
            if(!finished[i]) {
                answer[n]=todo_list[i];
                n++;
            }
        }
    return answer;
}

File: 프로그래머스/0/181885. 할 일 목록/할 일 목록.cs
using System;

public class Solution {
    public string[] solution(string[] todo_list, bool[] finished) {
        int count=0;
        for(int i=0; i<finished.Length ; i++){
            if(!finished[i]) {
                count++;
            }
        }
        string[] answer = new string[count];
        int n=0;
        for(int i=0; i<todo_list.Length ;i++){
            if(!finished[i]) {
                answer[n] = todo_list[i];
                n++;
            }
        }
        return answer;
    }
}